### PR TITLE
Footer-section

### PR DESCRIPTION
**Pull request by bruk19**
1. Added index.html for contact form.
2. Added style.css for contact design.
3. Added important icons.

### CODE REVIEW

**Review by OLIPLICHE — COMMENTED** [review]
Hi @bruk19 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have "node modules" folder inside `gitignore` file. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
...
         </section>
+        <footer class="contact">
+            <h2>I'm always <br> interested in hearing about new projects, 
+                so if you'd like to chat please get in</h2>
+            <div class="ip-type">
+                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
+                <input class="email" type="email" placeholder="   Email address" required>
+                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here"></textarea>
+                <button class="get" type="submit">Get in touch</button>
+            </div>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly put the contact form inside a form tag
- [ ] Kidly make sure that your form submit to [formspree service](https://formspree.io/html)

File: index.html [hunk 2]
```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
...
+                <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href="#"><i class="fa-solid fa-m"></i></a></li>
+            </ul>
+            <hr class="last">
+            </div>
+        </footer>
     </main>
 </body>
 </html>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly make sure your website is fully responsive on a mobile version

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @bruk19, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
                 </div>
             </div>
         </section>
+        <footer class="contact">
+            <h2>I'm always <br> interested in hearing about new projects, 
+                so if you'd like to chat please get in</h2>
+            <form class="ip-type" action="https://formspree.io/f/xzbokgee">
```
Comment (Ismailco) [inline index.html]: - [ ] Your not using a method for this form, kindly add a ```POST``` method to your form so it sends data to the form spree service

File: index.html [hunk 4]
```diff
@@ -164,6 +164,27 @@ <h3 class="skill">Skills</h3>
...
+                <input class="Name" type="text" placeholder="    Full Name" maxlength="30" required>
+                <input class="email" type="email" placeholder="   Email address" required>
+                <textarea name="msgt1" class="t1" cols="30" rows="10" maxlength="500" placeholder="  Enter text here"></textarea>
```
Comment (Ismailco) [inline index.html]: - [ ] The ```textarea``` doesn't have a required attribute, kindly add ```required``` to this input. it's a requirement

File: style.css [hunk 5]
```diff
@@ -487,3 +487,104 @@ hr {
   color: #fff;
   margin-top: 15px;
 }
+
+.contact {
```
Comment (Ismailco) [inline style.css]: - [ ] [OPTIONAL] The page is not fully mobile responsive, kindly fix it

File: style.css [hunk 6]
```diff
@@ -487,3 +487,104 @@ hr {
...
+
+.t1 {
+  height: 114px;
+  background: #fbfbf7;
+  border: none;
+  text-justify: top;
+  font-weight: bolder;
+}
+
+.get {
```
Comment (Ismailco) [inline style.css:540]: - [ ] Button interactions such as hover, active (when it's pressed), disabled, and enable (default state) are not styled as shown in the Figma template, please implement this so your design can be as close to the provided design as possible

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi  @bruk19,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @bruk19 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Implemented previous code review requested changes
- ✔️ descriptive commit messages

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [@mrramoun](https://github.com/mrramoun)
- This is My **Twitter**: [@ramoun16](https://twitter.com/ramoun16)
- This is My **Slack**: [Ramoun](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**